source/extensions/tracers/opentelemetry/grpc_trace_exporter.h

```diff
@@ -68,7 +68,7 @@ class OpenTelemetryGrpcTraceExporterClient : Logger::Loggable<Logger::Id::tracin
       if (stream_->stream_->isAboveWriteBufferHighWatermark()) {
         return false;
       }
-      stream_->stream_->sendMessage(request, false);
+      stream_->stream_->sendMessage(request, true);
```

Comment:
Can you add a test?

Reply:
Sure, I would be glad to do that, but I'm not sure how to write a new one that isn't covered already. The change in this pull request did cause [this test](https://github.com/envoyproxy/envoy/pull/29011/files#diff-99186a38c46cb43f8ffcd0740d86585cff78165f8192403b6852fca3ae7180daR44) to fail locally for me, but fixing the boolean causes it to pass again.

Is there any additional functionality you'd like to see tested? I'd be happy to do so with a bit of guidance!

Reply:
Maybe this is fine, but at least rename things like ` expectStreamMessage(` and the other stream stuff in those tests, the whole points is it's no longer a stream.

Reply:
Okay, I went ahead and renamed a few functions and variables. Would you mind taking a look again? Thanks!